builtin/locale/__builtin.ru.tr

```diff
@@ -0,0 +1,246 @@
+Rollback functions are disabled.=Функции отката отключены.
+That limit is too high!=Это <ограничение> слишком велико!
+Checking @1 ...=Проверка @1 ...
+Nobody has touched the specified location in @1 seconds.=Никто не прикасался к указанному местоположению в течение @1 секунд(ы).
+@1 @2 @3 -> @4 @5 seconds ago.=@1 @2 @3 -> @4 @5 секунд назад.
+Punch a node (range@=@1, seconds@=@2, limit@=@3).=Ударьте по ноде (диапазон@=@1, секунды@=@2, предел @=@3).
+(<name> [<seconds>]) | (:<actor> [<seconds>])=(<имя> [<секунды>]) | (:<игрок> [<секунды>])
+Revert actions of a player. Default for <seconds> is 60. Set <seconds> to inf for no time limit=Отменяет действия игрока. Значение по умолчанию для <секунд> равно 60. Установите <секунды> в значение inf для отключения ограничения по времени
+Invalid parameters. See /help rollback and /help rollback_check.=Недопустимые параметры. Смотрите /help rollback и /help rollback_check.
+Reverting actions of player '@1' since @2 seconds.=Отмена действия игрока "@1" за последние @2 секунд(у/ы).
+Reverting actions of @1 since @2 seconds.=Отмена действий @1 за последние @2 секунд(у/ы).
+(log is too long to show)=(журнал слишком длинный для отображения)
+Reverting actions succeeded.=Отмена действий завершилась успешно.
+Reverting actions FAILED.=Отменить действия НЕ УДАЛОСЬ.
+Show server status=Показать статус сервера
+This command was disabled by a mod or game.=Эта команда была отключена модом или игрой.
+[<0..23>:<0..59> | <0..24000>]=[<0..23>:<0..59> | <0..24000>]
+Show or set time of day=Показать или установить время суток
+Current time is @1:@2.=Текущее время @1:@2 .
+You don't have permission to run this command (missing privilege: @1).=У вас нет разрешения на запуск этой команды (отсутствует привилегия: @1).
+Invalid time (must be between 0 and 24000).=Недопустимое время (должно быть в диапазоне от 0 до 24000).
+Time of day changed.=Время суток изменено.
+Invalid hour (must be between 0 and 23 inclusive).=Недопустимый час (должно быть от 0 до 23 включительно).
+Invalid minute (must be between 0 and 59 inclusive).=Недопустимая минута (должно быть от 0 до 59 включительно).
+Show day count since world creation=Показать количество дней с момента сотворения мира
+Current day is @1.=Текущий день: @1.
+[<delay_in_seconds> | -1] [-r] [<message>]=[<задержка_в_секундах> | -1] [-r] [<сообщение>]
+Shutdown server (-1 cancels a delayed shutdown, -r allows players to reconnect)=Завершение работы сервера (-1 отменяет отложенное завершение работы, -r позволяет игрокам повторно подключиться)
+Server shutting down (operator request).=Завершение работы сервера (по запросу администрации).
```

Comment:
It can be several people on the server with the rights to do this. `Operator` sounds strange in ru, but `администратор` assumes a single (and technical) person (in ru for MT). So I decide to use plural `администрации`

Reply:
Alright